products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,274 @@
+The label `season` will be in string format and label `year` in number format.
+
+Till this point we only described nodes. Now we need to describe how those nodes are connected.
+
+* An edge of type `: TRANSFERRED_FROM` pointing from `Team` player is being transferred from
+to the node `Transfer`. Also, we add an edge of type `: TRANSFERRED_TO` pointing from the `Transfer`
+to the `Team` player is being transferred to.
+* From node `Player` we add an edge of type `: TRANSFERRED_IN` pointing to `Transfer`. 
+* Each transfer is made in a specific season, so we create
+edge  `: HAPPENED_IN` pointing from the node
+`Transfer` to the node `Season`.
+
+
+![](../data/FootballTransfers_metagraph.png)
+
+### Importing the Snapshot
+
+We have prepared a database snapshot for this example, so the user can easily
+import it when starting Memgraph using the `--data-directory` option.
+
+```plaintext
+sudo -u memgraph \
+  /usr/lib/memgraph/memgraph --data-directory /usr/share/memgraph/examples/TEDTalk \
+    --storage-snapshot-interval-sec=0 --storage-wal-enabled=false \
+    --storage-snapshot-on-exit=false --storage-properties-on-edges=true
+```
+
+When using Memgraph installed from a Debian or RPM package, 
+the currently running Memgraph server needs to be stopped before 
+importing the example, using the following command:
+
+```plaintext
+systemctl stop memgraph
+```
+
+When using Docker, the example can simply be imported with the following command:
+
+```plaintext
+docker run -p 7687:7687 \
+  -v mg_lib:/var/lib/memgraph -v mg_log:/var/log/memgraph -v mg_etc:/etc/memgraph \
+  memgraph --data-directory /usr/share/memgraph/examples/TEDTalk \
+  --storage-snapshot-interval-sec=0 --storage-wal-enabled=false \
+  --storage-snapshot-on-exit=false --storage-properties-on-edges=true
+```
+
+The user should note that any modifications of the database state will persist
+only during this run of Memgraph.
+
+### Example Queries using OpenCypher
+
+In the queries below, we are using [OpenCypher](https://www.opencypher.org) 
+to query Memgraph via the console.
+
+
+1) Let's say you want to find 20 most expensive transfers.
+As mentioned before fee is represented in millions of euros.
+
+```opencypher
+MATCH (t:Transfer)<-[:TRANSFERRED_IN]-(p:Player)
+WHERE t.fee is NOT NULL
+RETURN  t.fee as transfer_fee ,p.name AS player_name
+ORDER BY t.fee DESC LIMIT 20
+```
+
+2) What about finding the most expensive transfer per season? 
+
+```opencypher
+MATCH path = (s:Season)<-[:HAPPENED_IN]-(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player)
+WHERE t.fee is NOT NULL
+WITH  s.name as season, MAX(t.fee) as maxFee
+RETURN  maxFee,season
+ORDER BY maxFee DESC
+```
+
+
+3)How about finding out for which teams has your favorite player played for?
+Instead of Sime Vrsaljko, you can instead put the name of your favorite player.
+
+```opencypher
+MATCH (p:Player)-[:TRANSFERRED_IN]->(t:Transfer)-[]-(m:Team)
+WHERE p.name="Sime Vrsaljko"
+WITH DISTINCT m
+RETURN m.name AS team_name
+```
```

Comment:
Add a bit of description below this query why are you using `-[]-`.

Reply:
:+1: